backend/lcfs/web/api/transfer/repo.py

```diff
@@ -143,7 +147,8 @@ async def add_transfer_history(self, transfer_id: int, transfer_status_id: int)
         """
         new_history_record = TransferHistory(
             transfer_id=transfer_id,
-            transfer_status_id=transfer_status_id
+            transfer_status_id=transfer_status_id,
+            user_profile=self.request.user
```

Comment:
seems to be an error when saving a record.
`'TransferRepository' object has no attribute 'request'`

probably just adding the `request` object to the init up top

Reply:
I updated the service to pass down the user_profile_id. We probably should avoid having the request injected into the repo layer, seems a bit anti-repository pattern.